### PR TITLE
closes 138 control hit plot

### PR DESCRIPTION
**Pull request by Tremirre**
Implemented stacking control for Hit Determination chart, user may now define the bounds for the fitted function
As a side effect it also shows the user initial lower/upper limit

tbd if that should impact how activity/r2 is calculated 

default bounds:
image

stacking applied:
image

modified TOP/BOTTOM is saved in the resulting CSV file
unstacking brings back original values

### CODE REVIEW

File: dashboard/pages/hit_validation/callbacks.py [hunk 1]
```diff
@@ -270,6 +300,8 @@ def on_selected_compound_changed(
             ]
         ),
         "graph": graph,
+        "top": entry["TOP"],
+        "bottom": entry["BOTTOM"],
```
Comment (azywot) [inline dashboard/pages/hit_validation/callbacks.py:304]: I'd set the precision to 3-4 decimal places here

**Review by azywot — APPROVED** [review]
LGTM, tested locally, great job! 👍👍
Added just one small comment

File: dashboard/pages/hit_validation/callbacks.py (hunk 1 above)
Comment (Tremirre) [reply dashboard/pages/hit_validation/callbacks.py:304]: set it to 5 for consistency

**Review by AndrzejKaj — APPROVED** [review]
Everything looks good

**Review by zuzg — APPROVED** [review]

**Pull request merged**